workflow/labels.md

```diff
@@ -11,6 +11,7 @@ In this part you will find a description of all the labels we use on issues.
 | ----------- | ---------------------------------------------------------------------------------------- |
 | Don’t merge | While this label is present this pull-request should not be merged                       |
 | Duplicate   | This issue already exists                                                                |
+| Master Epic | This is a master [ZenHub epic](https://www.zenhub.com/blog/working-with-epics-in-github/)|
```

Comment:
Is it worth explaining what a master epic is a bit more? Or linking to a page that explains the concept?